components/post_view/post_list.jsx

```diff
@@ -269,7 +269,7 @@ export default class PostList extends React.PureComponent {
             return;
         }
 
-        if (this.state.isDoingInitialLoad || this.state.loadingPosts) {
+        if (this.state.isDoingInitialLoad || this.loadingPosts) {
```

Comment:
Don't you need to define and/or initialize `this.loadingPosts`, maybe at the constructor?

Reply:
Done. Added a default value